packages/ui/src/components/Accordion/StandardHeader.tsx

```diff
@@ -0,0 +1,67 @@
+import { Accordion } from "@gc-digital-talent/ui";
+import { AccordionHeaderProps } from "@gc-digital-talent/ui/src/components/Accordion/Accordion";
```

Comment:
We will need to update this import path so the package isn't importing itself.

```tsx
import { Accordion } from "./Accordion";
```

Though, i'm wondering if it is worth just moving this to `Accordion.tsx` and exporting it as part of the `Accordion` object? So we could do: `<Accordion.StandardHeader>`. But that can be saved for #6412

Reply:
Good catch!  I've fixed the imports in [fix imports](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6384/commits/9c2a04f5e52f8a7437946ebc7664299e4f79ecca).

Yeah, I'll consider exporting the StandardHeader in the Accordion in the main PR, after seeing how the new ones integrates.   I'm not quite comfortable with the name though.  Is there a word for a header and trigger together?  I don't want to call it StandardHeaderAndTrigger.  :laughing:

Reply:
heh, yeah... That is exactly what I was thinking. We could just make it obvious and say `Accordion.HeaderTrigger` 😅